msteams-platform/messaging-extensions/build-bot-based-agent.md

```diff
@@ -112,14 +112,14 @@ To trigger the message extension as agent in Microsoft 365 Copilot in Teams, fol
 
 1. Select **Apps**.
 1. Search for **Copilot** and open Microsoft 365 Copilot.
-1. Select **Plugins**.
-1. From the list of plugins, turn on the toggle for your message extension.
+1. Select **Panel** from the top-right corner.
+1. Select the required agent from the right-rail.
 
-   :::image type="content" source="../assets/images/Copilot/bot-based-VSC-plugin-toggle.png" alt-text="Screenshot shows the Plugin option, list of plugins, and the toggle enabled for bot-based-ME-test-local plugin.":::
+   :::image type="content" source="../assets/images/messaging-extension/message-extension-agents.png" alt-text="Screenshot shows the list of agents for bot-based-ME-test-local agent." lightbox="../assets/images/messaging-extension/message-extension-agent.png":::
```

Comment:
This screenshot doesn't match the app you are building. This sample agent is called npmsearch that allows a user to search npm packages, but in this screenshot is showing Contoso app and confuses readers.

Reply:
Updated the content for Contoso.

Reply:
How does the npm search sample app (ME) become Contoso agent, when running on Copilot? It doesn't make sense to me.

The feature is not rolled out to me yet, so I can't confirm this or take the correct screenshot for you, but I don't think it makes sense that the sample agent becomes the Contoso app all of the sudden. It should be still the same npm search agent. 
Please check and confirm.

Reply:
Yes @girliemac , we have used the updated DA UI screenshot provided by the PM. Since we couldn't test and capture the screenshot for the SBS guide with the NPM package as the feature will be fully rolled out next week.
 
For your information, I will update the SBS guide with the NPM package screenshot as soon as it is rolled out as a follow-up task (on 12/18).